services/federator/test/unit/Test/Federator/ExternalServer.hs

```diff
@@ -36,7 +36,7 @@ import Test.Tasty (TestTree, testGroup)
 import Test.Tasty.HUnit (assertEqual, testCase)
 import Wire.API.Federation.GRPC.Types
 
-genMock ''Brig
+genMock ''Service
 
 tests :: TestTree
 tests =
```

Comment:
Perhaps at some point it would make sense to add tests for Galley as well. If that makes sense, I'm fine with doing that in a separate PR.

Reply:
I added one test for galley.